presentation/src/main/java/com/example/healthysalad/OrderSummaryFragment.kt

```diff
@@ -35,27 +39,25 @@ class SelectBaseFragment : Fragment() {
         inflater: LayoutInflater, container: ViewGroup?,
         savedInstanceState: Bundle?
     ): View? {
-        //Data().loadItem("base")
-        // Inflate the layout for this fragment
-        var v : View = inflater.inflate(R.layout.fragment_select_base,container, false)
+        val view : View = inflater.inflate(R.layout.fragment_order_summary, container,false)
+        val viewManager: RecyclerView.LayoutManager = LinearLayoutManager(view.context)
+        val viewAdapter: OrderSummaryViewAdapter = OrderSummaryViewAdapter(Order)
+        val recyclerView : RecyclerView = view.findViewById(R.id.order_summary_recycler_view)
 
-        val openBackFrag: Button = v.findViewById(R.id.back_button)
-        openBackFrag.setOnClickListener {
-            activity!!.supportFragmentManager
-                .beginTransaction().replace(R.id.activity_main, TitleFragment())
-                .addToBackStack(null).commit()
-        }
-
-        val openNextFrag: Button = v.findViewById(R.id.next_button)
-        openNextFrag.setOnClickListener{
-            activity!!.supportFragmentManager
-                .beginTransaction().replace(R.id.activity_main, SelectProteinFragment())
-                .addToBackStack(null).commit()
-        }
+        recyclerView.setHasFixedSize(true)
+        recyclerView.layoutManager = viewManager
+        recyclerView.adapter = viewAdapter
 
 
-        return v
-
+        val openBackFrag: Button = view.findViewById(R.id.back_button)
+        openBackFrag.setOnClickListener {
+                activity!!.supportFragmentManager
+                    .beginTransaction().replace(
+                        R.id.activity_main, SelectIngredientFragment.newInstance("soft") // TODO: make select ingredient fragment scheduler be able to be accessed
```

Comment:
The `soft` string could be made as a constant, since it's used in several places. 

The reason is that if we were to change the text we would need to search for all the places this text is being used.